src/lib/structures/ModerationMonitor.ts

```diff
@@ -127,13 +128,27 @@ export abstract class ModerationMonitor<T = unknown> extends Monitor {
 	}
 
 	protected abstract keyEnabled: string;
+	protected abstract ignoredRolesPath: string;
```

Comment:
Can't you technically set these to `GuildSettings.Selfmod.IgnoreChannels`? Or a way to extract their type?